docs/tutorials/integrations/kserve/canary.md

```diff
@@ -225,7 +225,7 @@ kubectl delete isvc/wisdom-1
 Delete routing:
 
 ```shell
-cat <<EOF | helm template routing --repo https://iter8-tools.github.io/iter8 routing-actions --version 0.1.5 -f - | kubectl delete -f -
+cat <<EOF | helm template routing --repo https://iter8-tools.github.io/iter8 routing-actions --version 0.18 -f - | kubectl delete -f -
```

Comment:
So we decided to have `routing-actions` match the minor version of Iter8?

Reply:
Yes, this is what @sriumcp also recommended.